prime-angular-frontend/src/app/core/services/http-logger.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+
+import { LogType } from '@core/models/log-type.enum';
+import { LoggerResource } from '@core/resources/logger-resource.service';
+
+import { AbstractLoggerService } from './abstract-logger.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HttpLoggerService extends AbstractLoggerService {
+  constructor(
+    private loggerResource: LoggerResource,
+  ) {
+    super();
+  }
```

Comment:
Would this be better named `WebApiLoggerService`?

Reply:
I have no reserves for either name. Both seem to get the point across. Is there a reason you feel WebApi is better suited?

Reply:
For `WebApiLoggerService`, I think it's more clear where the logs are going to.  E.g. `ConsoleLoggerService`, the logs are going to the browser console